stac_api/runtime/tests/conftest.py

```diff
@@ -152,38 +198,6 @@
                     ]
                 ],
             },
-            "proj:projjson": {
-                "id": {"code": 4326, "authority": "EPSG"},
-                "name": "WGS 84",
-                "type": "GeographicCRS",
-                "datum": {
-                    "name": "World Geodetic System 1984",
-                    "type": "GeodeticReferenceFrame",
-                    "ellipsoid": {
-                        "name": "WGS 84",
-                        "semi_major_axis": 6378137,
-                        "inverse_flattening": 298.257223563,
-                    },
-                },
-                "$schema": "https://proj.org/schemas/v0.7/projjson.schema.json",
-                "coordinate_system": {
-                    "axis": [
-                        {
-                            "name": "Geodetic latitude",
-                            "unit": "degree",
-                            "direction": "north",
-                            "abbreviation": "Lat",
-                        },
-                        {
-                            "name": "Geodetic longitude",
-                            "unit": "degree",
-                            "direction": "east",
-                            "abbreviation": "Lon",
-                        },
-                    ],
-                    "subtype": "ellipsoidal",
-                },
-            },
```

Comment:
I had to remove this in order to get the tests to pass (failures are viewable here https://github.com/NASA-IMPACT/veda-backend/actions/runs/17807450210/job/50622755224) I believe these are not related to my changes since this is happening generally on other PRs (see: https://github.com/NASA-IMPACT/veda-backend/pull/529)

Reply:
I am pretty sure that failed action is the pgstac bug that we can fix with an upgrade https://github.com/NASA-IMPACT/veda-backend/issues/527.